src/server/connection.py

```diff
@@ -33,6 +37,8 @@ def __init__(
         server_port: int,
         on_connect: OnConnectFunc,
         log: logging.Logger,
+        is_outbound: bool = False,
+        is_feeler: bool = False,
```

Comment:
What is is_feeler? Can you add a comment?

Reply:
it's being explained in start_service.py.

Reply:
Consider removing these default `False` values. Also, I'm with Mariano... at least put in a comment pointing to the other comment.